packages/design-picker/src/components/index.tsx

```diff
@@ -223,30 +220,31 @@ const DesignPicker: React.FC< DesignPickerProps > = ( {
 	theme = 'light',
 	className,
 	highResThumbnails = false,
-	showCategoryFilter = false,
-	selectedCategory = null,
-	onSelectedCategoryChange = () => undefined,
-	showAllFilter = false,
 	categoriesHeading,
+	categorization,
 } ) => {
-	const categories = useCategorization( designs, showAllFilter );
+	const filteredDesigns = useMemo(
+		() => {
+			const result = ! categorization
```

Comment:
I think we only need to check `categorization?.selection` here and then we can remove the `!! categorization` from the depedencies.

Reply:
Cool call

Reply:
Great! This suggestion fixed the exhaustive deps warning so I was able to re-enable it 🙌